pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -337,6 +339,23 @@ func Newk8sOrchestrator(ctx context.Context, controllerClusterFlavor cnstypes.Cn
 	return k8sOrchestratorInstance, nil
 }
 
+func getReleasedVanillaFSS() map[string]struct{} {
```

Comment:
Can this is renamed to something like `getEnabledFeatureStates`  or `getEnabledFeatureGates`